mmv1/products/privateca/CertificateAuthority.yaml

```diff
@@ -136,8 +136,9 @@ virtual_fields:
     name: 'deletion_protection'
     default_value: true
     description: |
-      Whether or not to allow Terraform to destroy the CertificateAuthority. Unless this field is set to false
-      in Terraform state, a `terraform destroy` or `terraform apply` that would delete the instance will fail.
+      Whether Terraform will be prevented from destroying the CertificateAuthority.
+      When a`terraform destroy` or `terraform apply` would delete the CertificateAuthority,
```

Comment:
What would you think about stating what happens for both true and false, similar to https://developers.google.com/style/api-reference-comments#parameters?